examples/solids/index.rst

```diff
@@ -340,14 +340,64 @@ Carrying through the differentiation :math:numref:`strain-energy-grad` for the m
    For example, if :math:`e_{ij} \sim 10^{-8}`, then naive computation of :math:`\bm I_3 - \bm C^{-1}` and :math:`\log J` will have a relative accuracy of order :math:`10^{-8}` in double precision and no correct digits in single precision.
    When using the stable choices above, these quantities retain full :math:`\varepsilon_{\text{machine}}` relative accuracy.
 
+.. admonition:: Mooney-Rivlin model
+   :class: dropdown
+
+   Constitutive models of rubber and other nearly-incompressible materials are often expressed in terms of invariants of an isochoric Cauchy-Green tensor :math:`\bar{\bm C} = J^{-2/3} \bm C`, typically defined as
+
+   .. math::
+      \begin{aligned}
+      \mathbb{\bar I_1} &= \operatorname{trace} \bm{\bar C} \\
+                        &= J^{-2/3} \operatorname{trace} \bm C \\
+                        &= J^{-2/3} \mathbb I_1 \\
+      \mathbb{\bar I_2} &= \frac 1 2 \Big( (\operatorname{trace} \bm{\bar C})^2 - \bm{\bar C} \!:\! \bm{\bar C} \Big) \\
+                        &= \frac 1 2 \Big( \mathbb{\bar I_1^2} - J^{-4/3} \bm C \!:\! \bm C \Big) \\
+                        &= J^{-4/3} \mathbb I_2
+      \end{aligned}
+
+   along with the determinant :math:`J = \sqrt{\lvert \bm C \rvert}`.
+
+   The Mooney-Rivlin strain energy density (cf. Neo-Hookean :math:numref:`neo-hookean-energy`) is :cite:`bower2010applied`
+
+   .. math::
+      :label: mooney-rivlin-energy
+
+      \Phi(\mathbb{\bar I_1}, \mathbb{\bar I_2}, J) = \frac{\mu_1}{2} (\mathbb{\bar I_1} - 3) + \frac{\mu_2}{2} (\mathbb{\bar I_2} - 3) + \frac{k_1}{2} (J - 1)^2,
+
+   which we differentiate as in the Neo-Hookean case :math:numref:`neo-hookean-stress` to yield the second Piola-Kirchoff tensor,
+
+   .. math::
+      :label: mooney-rivlin-stress
+
+      \bm S = \mu_1 \mathbb I_1 J^{-2/3}(\bm I_3 - \frac{1}{3} \bm C^{-1}) + \mu _2 I_2J^{-4/3}(I_1 - \bm C - \frac{2}{3} \bm C^{-1}) + k_1(J^2 -J)\bm C^{-1} ,
+
+   [TODO: double-check above] where we have used
```

Comment:
I didn't mess with this equation, but if the new text above and below is correct, it'll need to be adjusted. (I may well have bungled something so please double-check me.)